lib/wallaroo/core/topology/router.pony

```diff
@@ -1416,82 +1774,78 @@ class val LocalPartitionRouter[In: Any val,
     end
     consume digest
 
+  fun state_entity_digest(): Array[String] val =>
+    // Return an array of keys
```

Comment:
// Return an array of String representation of keys for use in query response?

Reply:
done